### ISSUE TITLE
Warnings on forward sync

### ISSUE THREAD

**Issue by ArchangeGabriel** [peder2tm/sved#7]
Each time I call `SVED_sync()` I get this displayed at the bottom of my vim:
```
/home/<user>/.vim/ftplugin/evinceSync.py:197: PyGIDeprecationWarning: GObject.MainLoop is deprecated; use GLib.MainLoop instead
  gobj_main_loop = GObject.MainLoop()
/home/<user>/.vim/ftplugin/evinceSync.py:212: PyGIDeprecationWarning: GObject.timeout_add is deprecated; use GLib.timeout_add instead
  GObject.timeout_add(10, sync_view, pdf, path_source, curpos)
/home/<user>/.vim/ftplugin/evinceSync.py:206: PyGIDeprecationWarning: GObject.timeout_add is deprecated; use GLib.timeout_add instead
  GObject.timeout_add(100, gobj_main_loop.quit)
```
Fixing seems pretty straight forward (I haven’t tried though).

**Comment by peder2tm** [comment]
Are you using the latest version?
This was fixed in commit 31c62f0b2b1ddd3e3b1b20506953e03df8217697

**Comment by ArchangeGabriel** [comment]
Interesting, for some reason I missed that commit. It obviously fixes the issue. :)
